stylesheets/global-style.css

```diff
@@ -0,0 +1,374 @@
+  flex-flow: column wrap;
+  float: right;
+  padding: 10px;
+}
+
+.button {
+  font: bold 11px Arial;
+  text-decoration: none;
+  background-color: #99CC33;
+  color: #333333;
+  border: 1px solid #CCCCCC;
+}
+
+.see-dogs {
+  font: bold 20px Times;
+  text-decoration: none;
+  background-color: #D0D0D0;
+  color: #2952a3;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.submit-button {
+  font: bold 20px Times;
+  text-decoration: none;
+  background-color: #99CC33;
+  color: white;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.navbar-container a {
+  font-family: 'Merriweather Sans', sans-serif;
+  font-size: 30px;
+}
+
+nav ul {
+  list-style-type: none;
+}
+
+nav ul li {
+	float: left;
+	color: #FFFFFF;
+	margin-right: 2%;
+	letter-spacing: 2px;
+}
+ul li a {
+  text-decoration: none;
+}
+
+.dog-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  float: left;
+  grid-gap: 30px;
+}
+
+.dog-box button {
+  background-color: #99CC33;
+  color: white;
+  font-style: oblique;
+  font-weight: bold;
+  text-align: center;
+}
+
+.dog-box {
+  padding: 0px;
+  border: 1px solid gray;
+  margin: 5%;
+  float: left;
+}
+
+.dog-box p {
+  padding-left: 10px;
+  text-align: left;
+}
+
+.dog-box h3 {
+  padding-left: 10px;
+  text-align: left;
+  margin-left: 2px;
+  color: #2952a3;
+}
+
+.dog-box h4 {
+  padding-left: 10px;
+  text-align: left;
+}
+
+.dog-box button {
+  display: inline-block;
+  width: 95%;
+  margin: 5px;
+}
+
+.hero {
+  width: 100%;
+  padding: 10px;
+  box-sizing: border-box;
+  overflow:hidden;
+  text-align: left;
+}
+
+.aside {
+  float: left;
+  width: 55%;
+}
+
+.hero-text {
+  float: left;
+  margin-left: 10px;
+}
+
+.blog-post {
+  width: 100%;
+  overflow:hidden;
+  text-align: left;
+}
+
+.image-section,
+.aside {
+  margin: 10px;
+}
+
+aside h2 {
+  color: #2952a3;
+}
+
+.image-section {
+  float: left;
+  width: 30%;
+  margin-left: 5%;
+  padding: 10px;
+}
+
+aside {
+  float: right;
+  width: 25%;
+  min-width: 10%;
+  padding: 20px;
+}
+
+.dogs-adopted {
+  width: 10%;
+  padding: 0px;
+  border: 1px solid gray;
+  box-sizing: border-box;
+  margin: auto;
+}
+
+form {
+  color: #2952a3;
+  font-weight: bold;
+}
+
+.text-input {
+  width: 50%;
+  height: 25px;
+}
+
+.checkout-dogs {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.footer {
+  background-color: #383838;
+  align-items: center;
+  color: white;
+  width: 100%;
+  clear: both;
+}
+
+.footer img {
+  align-items: center;
+  width: 30%;
+  padding-top: 3%;
+}
+
+.footer a {
+  font-size: 25px;
+  color: white;
+  font-variant: small-caps;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.footer p {
+  padding-bottom: 3%;
+}
+
+h1 {
+  color: #2952a3;
+}
+
+main {
+  max-width: 70%;
+  float: left;
+}
+
+/* for-phone-only */
+@media (max-width: 599px) {
+
+  .dog-container,
+  .dog-box {
```

Comment:
Your .dog-box is the actual dog cards and needs no css grid layout, the layout should be applied to the container only not the children, this seems to be having no additional effect.